cfg/config.go

```diff
@@ -563,6 +565,12 @@ func BuildFlagSet(flagSet *pflag.FlagSet) error {
 
 	flagSet.DurationP("http-client-timeout", "", 0*time.Nanosecond, "The time duration that http client will wait to get response from the server. A value of 0 indicates no timeout.")
 
+	flagSet.IntP("http-dns-cache-ttl-secs", "", 0, "Sets the DNS cache TTL to the specified value. Special values: -1 means infinite TTL and 0 disables caching")
```

Comment:
can you make it clear that it only applies to http1? maybe just name it http1-dns-cahce-ttl-secs